wire-library/wire-runtime/src/commonMain/kotlin/com/squareup/wire/WireField.kt

```diff
@@ -44,7 +45,13 @@ annotation class WireField(
   /**
    * Redacted fields are omitted from toString() to protect sensitive data. Defaults to false.
    */
-  val redacted: Boolean = false
+  val redacted: Boolean = false,
+  /**
+   * Original name of this field as defined in the proto schema. This value is set to a non-empty
+   * string only when the original name differs from the generated one; for instance, a proto field
+   * named `object` generated in Java will be renamed `object_`.
```

Comment:
Is this actually the case with JavaGenerator (since `object` is not a Java keyword)?

Reply:
Java shouldn't renamed `object` no

Reply:
oh I see...

Reply:
I can submit a PR.